### ISSUE TITLE
make documentation reproducible

### ISSUE THREAD

**Issue by umlaeute** [csound/csound#1525]
currently, the generated Doxygen documentation includes the full paths to the referenced files.
This means, that depending on the filesystem layout of the build machine (that is: depending on the directory where the CSound sources are unpacked), the documentation files will be slightly different.

In the spirit of the [Reproducible Builds Effort](https://reproducible-builds.org/), I would change the Doxygen configuratoin to *not* use `FULL_PATH_NAMES`:

~~~diff
--- csound.orig/doc/Doxyfile
+++ csound/doc/Doxyfile
@@ -140,7 +140,7 @@
 # shortest path that makes the file name unique will be used
 # The default value is: YES.
 
-FULL_PATH_NAMES        = YES
+FULL_PATH_NAMES        = NO
 
 # The STRIP_FROM_PATH tag can be used to strip a user-defined part of the path.
 # Stripping is only done if one of the specified strings matches the left-hand
--- csound.orig/doc/Doxyfile-CsoundAC
+++ csound/doc/Doxyfile-CsoundAC
@@ -140,7 +140,7 @@
 # shortest path that makes the file name unique will be used
 # The default value is: YES.
 
-FULL_PATH_NAMES        = YES
+FULL_PATH_NAMES        = NO
 
 # The STRIP_FROM_PATH tag can be used to strip a user-defined part of the path.
 # Stripping is only done if one of the specified strings matches the left-hand
~~~

We are using this patch for the Debian packages.

**Comment by vlazzarini** [comment]
CsoundAC is not part of Csound anymore and so the second file will be removed. The patch is implemented on edd98171af6213b03eb55ee26b389caae05b284b

**Comment by vlazzarini** [comment]
with thanks
